todo_app/templates/index.html

```diff
@@ -2,35 +2,75 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+<header>
+  <style type="text/css">
+    .collumn {
+      float: left;
+      width: 33%;
+    }
+  </style>
+</header>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-        {% for to_do in to_dos %}
-        <li>
-        <h1><a href="/to_do/{{to_do.id}}">{{to_do.id}}</a> {{to_do.title}}</h1>
-        <p>{{to_do.status}}</p>
-        </li>
+<div class="jumbotron">
+  <h1 class="display-4">To-Do App</h1>
+  <p class="lead">Just another to-do app.</p>
+</div>
 
+<div class="row">
+  <div class="col-md-12">
+    <div class="row">
+      <div class="collumn">
+        <h2>Items</h2>
+      </div>
+      <div class="collumn">
+        <form action="/add_item" method="post">
+          <div>
+            <input type="text" name="title" id="title" value="Add new item" required>
```

Comment:
Placeholders disappear when you start typing in a text box, which is what we want here.
```suggestion
            <input type="text" name="title" id="title" placeholder="Add new item" required>
```